chandra_aca/tests/test_star_probs.py

```diff
@@ -73,14 +135,18 @@ def test_acq_success_prob_color():
     HAS_AGASC = True
 except:
     HAS_AGASC = False
+
+
 @pytest.mark.skipif('not HAS_AGASC', reason="Test requires AGASC")
-def acq_success_prob_from_stars():
+def test_acq_success_prob_from_stars():
     # These are acq stars for obsid 20765
-    star_ids = [118882960, 192286696, 192290008, 118758568, 118758336, 192291664, 192284944, 192288240]
+    star_ids = [118882960, 192286696, 192290008, 118758568,
+                118758336, 192291664, 192284944, 192288240]
     hws = [160, 160, 120, 160, 120, 120, 120, 120]
     stars = [agasc.get_star(agasc_id) for agasc_id in star_ids]
     mags = [star['MAG_ACA'] for star in stars]
     colors = [star['COLOR1'] for star in stars]
-    probs = acq_success_prob(date='2018:059', t_ccd=-11.2, mag=mags, color=colors, halfwidth=hws)
-    assert np.allclose(probs, [0.978, 0.967, 0.801, 0.755, 0.575, 0.089, 0.682, 0.659],
```

Comment:
There might have been value in leaving the old test too, but I could go either way on that.

Reply:
When I looked at this I decided that 2018:059 is firmly in the realm where the spline model is much more relevant.  But this makes we wonder if I shouldn't set the transition date to be more like 2017-Oct when we had the limit increase and have actual data saying the SOTA model is not fitting.  So use a data-driven cut-off instead of a release-date-driven cut-off.  Thoughts?

Reply:
And my thought from a code-validation perspective was that you've already got some outputs here and it is easy to keep them for validation.  I probably would have kept the old model test but called it by specifying 'sota' instead of letting it decide based on date (though there probably should be a test that decides based on date so you actually get test coverage of that code).

With regard to the actual time of transition, I don't think it matters.  Not sure what "have actual data saying the SOTA model is not fitting" means in your context.  For applications like starcheck, we're only going to care about the results going forward (so any date works).  For applications where you are looking at model performance over a chunk of time, you are going to be specifying which model to use.  For applications like the aca_lts_eval code, I only care about being able to call the sota model if we want to make a new comparison, and for that, I'll be specific.